spring-cloud-skipper/src/main/java/org/springframework/cloud/skipper/domain/ScaleRequest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.skipper.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Janne Valkealahti
+ */
+public class ScaleRequest {
```

Comment:
Deployer's scale domain time take deployerId, count and optional Map<Sting, String>. 
To represent this on the Skipper side the `ScaleRequest` class, perhaps can have a structure something like this: 
```java
public class ScaleRequest {
   private String name; // appName (not deployerId)
   private String count; // instance count
   private Map<String, String> properties; // optional
   // ... Getters/Setters ...
}
```
And then to represent multiple app scales we can use `List<ScaleRequest>`. Above should match to JSON like:

```json
[ 
 {"name": "time", "count": "2", "properties": {...}},
 {"name": "log", "count": "3", "properties": {...}},
 ...
]
```

Reply:
done